docs/get_started.ipynb

```diff
@@ -1,1074 +1,1656 @@
+    "    transform data by converting the *preprocessing function* into a *Beam\n",
+    "    pipeline*.\n",
+    "*   Show additional usage examples."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "9_SoiTcNmkVu"
+   },
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "gc6oSu9BnwJe"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -U tensorflow_transform"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "pY_BNfLemjY4"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install pyarrow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "L7Mtis2Jn2Af"
+   },
+   "outputs": [
     {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "372894b6"
-      },
-      "outputs": [],
-      "source": [
-        "from tensorflow_transform.tf_metadata import dataset_metadata\n",
-        "from tensorflow_transform.tf_metadata import schema_utils\n",
-        "\n",
-        "raw_data_metadata = dataset_metadata.DatasetMetadata(\n",
-        "      schema_utils.schema_from_feature_spec({\n",
-        "        's': tf.io.FixedLenFeature([], tf.string),\n",
-        "        'y': tf.io.FixedLenFeature([], tf.float32),\n",
-        "        'x': tf.io.FixedLenFeature([], tf.float32),\n",
-        "    }))"
+     "data": {
+      "text/plain": [
+       "<module 'pkg_resources' from '/home/pdodeja/.pyenv/versions/3.8.5/lib/python3.8/site-packages/pkg_resources/__init__.py'>"
       ]
-    },
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pkg_resources\n",
+    "import importlib\n",
+    "importlib.reload(pkg_resources)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "id": "PvDoWUfynTWh"
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import tempfile\n",
+    "\n",
+    "import tensorflow as tf\n",
+    "import tensorflow_transform as tft\n",
+    "import tensorflow_transform.beam as tft_beam\n",
+    "\n",
+    "from tensorflow_transform.tf_metadata import dataset_metadata\n",
+    "from tensorflow_transform.tf_metadata import schema_utils\n",
+    "\n",
+    "from tfx_bsl.public import tfxio"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "j4W_yuSr-ro3"
+   },
+   "source": [
+    "## Define a preprocessing function\n",
+    "\n",
+    "The *preprocessing function* is the most important concept of `tf.Transform`.\n",
+    "The preprocessing function is a logical description of a transformation of the\n",
+    "dataset. The preprocessing function accepts and returns a dictionary of tensors,\n",
+    "where a *tensor* means `Tensor` or `SparseTensor`. There are two kinds of\n",
```

Comment:
Shouldn't this be three kinds?

Reply:
@rcrowe-google you are right.  I will correct this and re-submit. Thank you for reviewing!